docs/snippets/change-a-currency-symbol.md

```diff
@@ -0,0 +1,28 @@
+# Rename a country
+
+See the [currency list](https://woocommerce.github.io/code-reference/files/woocommerce-includes-wc-core-functions.html#source-view.475) for reference on currency codes.
+
+
+> This is a **Developer level** doc. If you are unfamiliar with code and resolving potential conflicts, select a [WooExpert or Developer](https://woocommerce.com/customizations/) for assistance. We are unable to provide support for customizations under our [Support Policy](http://www.woocommerce.com/support-policy/).
+
+Add this code to your child theme’s `functions.php` file or via a plugin that allows custom functions to be added, such as the [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin. Avoid adding custom code directly to your parent theme’s functions.php file, as this will be wiped entirely when you update the theme.
+
+```php
+if ( ! function_exists( 'YOUR_PREFIX_change_currency_symbol' ) ) {
+  /**
+   * Change a currency symbol
+   * 
+   * @param string $currency_symbol Existing currency symbols.
+   * @param string $currency Currency code.
+   * @return string $currency_symbol Updated currency symbol(s).
+   */  
+  function YOUR_PREFIX_change_currency_symbol( $currency_symbol, $currency ) {
+    switch( $currency ) {
```

Comment:
Optional thing, since I think the goal is to port docs across rather than finesse them, but we could tweak the spacing here:

```suggestion
    switch ( $currency ) {
```

Reply:
Good catch. Seems my PHPCS was failing here. I'll double-check this for future PRs.